webpack/assets/javascripts/react_app/common/__mocks__/I18n.js

```diff
@@ -7,11 +7,19 @@ export const ngettext = createTranslateMock();
 
 export const sprintf = createTranslateMock();
 
+export const intl = {
+  loaded: true,
+  ready: Promise.resolve(),
```

Comment:
Maybe removing `loaded` and `Promise.resolve(true)` could fix the failing snapshot tests...